app/templates/new-platform-flexberry-g-i-s-layer-metadata-e.hbs

```diff
@@ -18,6 +18,7 @@
     onShow=(action "onErrorMessageShow")
```

Comment:
This `ui-message` became unnecessary now, and should be removed with it's action handlers.